Storage/SQL/SQLiteHistory.swift

```diff
@@ -586,7 +586,10 @@ extension SQLiteHistory: BrowserHistory {
         ") ORDER BY frecency DESC" +
         " LIMIT 1000"                                 // Don't even look at a huge set. This avoids work.
 
-        // Next: merge by domain and sum frecency, ordering by that sum and reducing to a (typically much lower) limit.
+        // Next: merge by domain and select the url with the max frecency of a domain, ordering by that sum frecency and reducing to a (typically much lower) limit.
+        // NOTE: When using GROUP BY we need to be explicit about which url to use when merging. By using "max(frecency)" the result row
+        //       for that domain will contain the url with the max frecency https://sqlite.org/lang_select.html#resultset
```

Comment:
Nit: s/with the/corresponding to the history item with the
Nit: colon after "frecency"